ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -395,18 +398,28 @@ class FamilyReads extends React.PureComponent {
     />
 
     const igvSampleIndividuals = this.state.openFamily && (igvSamplesByFamilySampleIndividual || {})[this.state.openFamily]
+    const dnaTrackOptions = DNA_TRACK_TYPE_OPTIONS.filter(({ value }) => igvSampleIndividuals && igvSampleIndividuals[value])
```

Comment:
it would be neater to add ` || {}` to the assignment for `igvSampleIndividuals` so you don't have to keep checking for null